style.css

```diff
@@ -410,67 +455,21 @@ body {
   justify-content: center;
 }
 
-.contact-container {
-  text-align: center;
-  padding-top: 80px;
-  color: white;
-}
-
-.inner-form-container {
-  padding: 0;
-}
-
-.contact-container h1 {
-  font-size: 40px;
-}
-
-.contact-container p {
-  font-size: 20px;
-  margin-left: 300px;
-  margin-right: 300px;
-}
-
-.form-control {
-  display: flex;
-  justify-content: center;
-}
-
-.form-control input,
-textarea {
-  margin-top: 10px;
-  border-radius: 5px;
-  padding: 10px;
-  border: none;
-  width: 100%;
-  display: block;
-}
-
-.btn-div {
-  text-align: center;
-}
-
-.submitBtn {
-  border: none;
-  padding: 10px;
-  background-color: white;
-  color: #6070ff;
-  transition: 0.5s;
-  margin-bottom: 50px;
-  margin-top: 10px;
-  border-radius: 5px;
-  cursor: pointer;
+.about-column {
+  flex: 1;
+  padding-right: 30px;
 }
 
-.submitBtn:hover {
-  background-color: #6070ff;
-  color: white;
+.about-column-text {
+  flex: 1;
+  padding-left: 30px;
 }
 
-.mobile_nav {
-  display: none;
-}
+/* desktop */
 
 @media only screen and (max-width: 767px) {
```

Comment:
As suggested by the previous reviewer,  I kindly suggest that you make your project styling mobile-first. That would mean that you need to use `min-width` to style for desktops and not `max-width` to style for mobile.

## EXAMPLE
`@media only screen and (min-width: 768px)  {...}
`